pkg/cmd/dev_test.go

```diff
@@ -0,0 +1,48 @@
+package cmd_test
```

Comment:
I'd prefer to keep it as an example test - both because it [runs as part of `go test`](https://pkg.go.dev/testing#hdr-Examples) because godoc renders it as a runnable example in the page, like https://pkg.go.dev/github.com/snyk/policy-engine@v0.21.1/pkg/input#example-Loader.Load

Typically I've kept these in the same place as the code that they're testing, but it could make sense to link to it from further up.

Reply:
TIL :)